### ISSUE TITLE
From datetime_select render ugly

### ISSUE THREAD

**Issue by JeromeDesseaux** [mkhairi/materialize-sass#86]
Hi :) 

First: thanks for this awesome gem! I noticed that datetime_select are rendering ugly on my pages (I can't use it or even select a single field).

I'm looking for a workaround or something :) 

Thanks for the help!

**Comment by mkhairi** [comment]
hi, @JeromeDesseaux, there is no exact workaround solution for this problem at this moment. If you use simple_form, you can build you own simple_form wrapper to fix this. see this [solution](http://stackoverflow.com/questions/29182590/datetime-select-styling-with-simple-form-bootstrap-and-rails-4)  but base on bootstrap framework.  i suggest, you should use materialize datepicker or other external  datetimepicker plugin  .

**Comment by lebart** [comment]
hi, 
For my datetime_selectors I use a combination of a date_selector with the date picker and a datetime_selector (basically it's just a dropdown select) with the option ignore date set to true. Then in the controller I grab the params and refactor them to make them the 'rails way' 

With slim it's looking like: 

       .input-field.col.s6
          =f.text_field :start, value: @activity.start.to_date, class: "datepicker"
          =f.label :start

        .input-field.col.s6.timeSelectorContainer
          =f.time_select :start, ignore_date: true
          =f.label :start, "HH:MM"

It looks like this

![capture d ecran 2016-02-15 a 11 58 01](https://cloud.githubusercontent.com/assets/2455364/13055114/756e9b8a-d3db-11e5-9460-09b5ccc6c5f1.png)

**Comment by mkhairi** [comment]
:+1:

**Comment by lebart** [comment]
Have you tried the suggested solution? 
If so can we close this issue ?

**Comment by JeromeDesseaux** [comment]
I've finally used a JQuery datetime selector. I can explain here how did I integrate it within my website if needed. If not, you can close the issue :) 

Thanks for the help :D

**Comment by AlexVentura** [comment]
I just have adjusted some params and I got something a little bit pretty by doing:

```
<div class="input-field col s12">
	<%= f.datetime_select :talk_schedule, { ampm: true, time_separator: '', start_year: DateTime.now.year, order: [:day, :month, :year] }, { class: 'datepicker col s2' } %>
	<%= f.label :talk_schedule, 'Talk schedule', class: 'active' %>
</div>
```

I got this:

<img width="540" alt="screen shot 2016-11-01 at 17 15 57" src="https://cloud.githubusercontent.com/assets/13169164/19911092/f158aff4-a056-11e6-8400-bc8e6cdfad50.png">

It may not be the better but it looks more usable.

BTW:
I don't know how to quit the `-` symbol which is the separator between Date-Time parts of the attribute. Any help will come in handy.
